packages/Tokens/src/tokens.yaml

```diff
@@ -148,6 +148,11 @@ popover:
   border-radius: $card--border-radius
   shadow: 0 2px 4px rgba(0, 0, 0, 0.1)
 #
+counter:
+  radius: 50px
```

Comment:
Oh, this isn't very clean, I don't feel great about tokenizing it, especially since it's not for the "default" (grey) counter. Maybe we can just just 100% or something inline to make it a round dot?